app/decorators/hearing_decorator.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class HearingDecorator < BaseDecorator
+  def provider_list
+    return t('generic.not_available') if providers.blank? || providers.empty?
+
+    safe_join(provider_sentences, tag.br)
+  end
+
+  def defendant_name_list
+    return t('generic.not_available') if defendant_names.blank? || defendant_names.empty?
+
+    safe_join(defendant_names, tag.br)
+  end
+
+  def prosecution_advocate_name_list
+    return t('generic.not_available') \
```

Comment:
I'd prefer this to be on a single line if possible.

Reply:
@jrmhaig me too, but then it would need to be excluded by rubocop or we allow longer lines in rubocop?!

Reply:
I thought I had written a comment elsewhere about not needing both `.blank?` and `.empty?` but I must not have submitted it. I've added it now (above).

Reply:
I see yes 👍